phpBB/styles/prosilver/template/smiley_dropdown.html

```diff
@@ -0,0 +1,40 @@
+    <div id="smiley-dropdown" class="dropdown dropdown-extended" >
+        <div class="pointer"><div class="pointer-inner"></div></div>
+        <div class="smiley-content">
+            <div id="smiley-box" class="smiley-box">
+                <ul>
+                    <!-- EVENT posting_editor_smilies_before -->
+                    <!-- IF S_SMILIES_ALLOWED and .smiley -->
+                    <!-- BEGIN smiley -->
+                    <li class="smiley-list">
+                        <a href="#" onclick="insertText('{smiley.A_SMILEY_CODE}')"><img src="{smiley.SMILEY_IMG}" width="{smiley.SMILEY_WIDTH}" height="{smiley.SMILEY_HEIGHT}" alt="{smiley.SMILEY_CODE}" title="{smiley.SMILEY_DESC}" /></a>
```

Comment:
yeah, I'm familiar with jquery and dome selection. Do you need me to insert a id attribute here and select that inside forum_fn.js using `$('#id').click(function(){
    /*function to run
});`

I used previous way because I thought it would be good to use the same way as it was.